codegen/package.json

```diff
@@ -8,6 +8,9 @@
   },
   "bugs": "https://github.com/whispir/openapi/issues",
   "scripts": {
+    "generate:java": "yarn generate --generator-key java --custom-generator ./generators/java/target/java-openapi-generator-1.0.0.jar",
```

Comment:
Per the update to [generate-sdk workflow](https://github.com/whispir/openapi/pull/176/files#diff-dda97367049910868155a384fc513c14c18b3cb6fec946b17c25ae549f9f9887R37), we need to add a `generate:node` script too